gapis/api/vulkan/replay.go

```diff
@@ -943,6 +943,7 @@ func (a API) Replay(
 			}
 			profile.AddResult(rr.Result)
 			optimize = false
+			// TODO(apbodnar) add the WaitForProfiler transform _somewhere_. Possibly here
```

Comment:
remove this comment, it's implied that we need to actually start using the WAIT operation.